src/trans/gpu/internal/trltog_mod.F90

```diff
@@ -121,7 +121,7 @@ SUBROUTINE TRLTOG(ALLOCATOR,HTRLTOG,PREEL_REAL,KF_FS,KF_GP,KF_UV_G,KF_SCALARS_G,
     USE ISO_C_BINDING,          ONLY: C_SIZE_T
     USE OPENACC_EXT,            ONLY: EXT_ACC_ARR_DESC, EXT_ACC_PASS, EXT_ACC_CREATE, &
       &                               EXT_ACC_DELETE
-    USE OPENACC,                ONLY: ACC_HANDLE_KIND
+    USE OPENACC,                ONLY: ACC_IS_PRESENT
```

Comment:
```suggestion
    USE OPENACC,                ONLY: ACC_HANDLE_KIND, ACC_IS_PRESENT
```

Reply:
Same thing as in the other file.

Reply:
I should have realized... sorry for that :)